src/automaton/core/data/protobuf/protobuf_factory.cc

```diff
@@ -251,7 +251,7 @@ void protobuf_factory::import_schema(schema* schema, const string& name, const s
   LOG(INFO) << "Importing schema " << name << " with package " << package;
   std::string json;
   schema->to_json(&json);
-  VLOG(9) << json;
+  // VLOG(9) << json;
```

Comment:
You may want to make these LOG(INFO) for now. Later we can add our own custom verbose levels.
In general it's not a good practice to check in commented out code.